infrastructure-playbooks/switch-from-non-containerized-to-containerized-ceph-daemons.yml

```diff
@@ -260,14 +260,15 @@
         find /var/lib/ceph/osd {% if dmcrypt | bool %}/var/lib/ceph/osd-lockbox{% endif %} -maxdepth 1 -mindepth 1 -type d
       register: osd_dirs
       changed_when: false
+      failed_when: false
 
     - name: unmount all the osd directories
       command: >
         umount {{ item }}
       changed_when: false
       failed_when: false
       with_items: "{{ osd_dirs.stdout_lines }}"
-      when: osd_running.rc != 0
+      when: osd_running.rc != 0 or osd_running.stdout_lines | length == 0
```

Comment:
do we really need the first part of the condition? given that we know it always returns 0

Reply:
yes because if docker/podman isn't installed then the rc code won't 0.

BTW the docker/podman installation is done after this part.

Reply:
makes sense then